crypto/init.c

```diff
@@ -343,6 +331,18 @@ DEFINE_RUN_ONCE_STATIC(ossl_init_engine_dynamic)
     return 1;
 }
 # ifndef OPENSSL_NO_STATIC_ENGINE
+# if !defined(OPENSSL_NO_DEVCRYPTOENG)
+static CRYPTO_ONCE engine_devcrypto = CRYPTO_ONCE_STATIC_INIT;
+DEFINE_RUN_ONCE_STATIC(ossl_init_engine_devcrypto)
+{
+#  ifdef OPENSSL_INIT_DEBUG
+    fprintf(stderr, "OPENSSL_INIT: ossl_init_engine_devcrypto: "
+                    "engine_load_devcrypto_int()\n");
+#  endif
+    engine_load_devcrypto_int();
+    return 1;
+}
+# endif
```

Comment:
Watch those preprocessor directive indents, they should all be pushed on space to the right.

Reply:
Fixed it.  Now I do have a question regarding these directives.  There is a mix of `#ifndef`s and `#if !defined`s here.  I followed the AFALG's lead and left it `!defined`.  Do you want me to change any of them?

Reply:
Ah, good question!  Different people do this with different styles, and we haven't been very strict.

I usually try to follow the style of already existing code in the file I make changes in, so with that in mind, checks of single macros would be in the `#ifdef` / `#ifndef` form.

Reply:
I've changed the `OPENSSL_NO_DEVCRYPTOENG` to `ifndef`s.  Should I change AFALG's, or the "don't touch anything unrelated to your change" has the priority here.  I'm assuming it does, so I'll leave it alone.  I'll give the CIs a rest and wait for more changes before I push the commit, but it is done.

Reply:
I think that AFALG changes should be for another PR, regardless of who does it.  Let's stay focused on devcrypto here, yeah?